src/scss-migrate/index.ts

```diff
@@ -56,10 +56,19 @@ export function scssMigrate(options: Schema): Rule {
       const lastPosOfPathDelimiter = defaultProjectPath.lastIndexOf('/');
       const srcRoot = defaultProjectPath.substr(0, lastPosOfPathDelimiter + 1);
 
-      tree.exists(`${srcRoot}/styles.${options.from}`) && tree.rename(
-        `${srcRoot}/styles.${options.from}`,
-        `${srcRoot}/styles.${options.to}`
-      );
+      // convert root styles.scss file content
+      if (options.from === 'scss' && options.to === 'css') {
+        const target = `${srcRoot}styles.scss`;
+        const data = tree.read(target).toString();
+        const result = renderSync({ data });
+        tree.create(`${srcRoot}styles.css`, result.css.toString());
+        tree.delete(`${srcRoot}styles.scss`);
```

Comment:
@d-koppenhagen  this seems to delete files without creating their corresponding .css files.
And the content of the deleted files is lost and not copied to the new .css file.
I have reverted this merge.
Please confirm this and create a new PR

Reply:
@Teebo did you run it locally or with a deployed new schematics version? When running the schematics locally against an angular workspace, everything seems to work and the content is not lost.
Did you run `npm i` before in case you ran it locally?

Reply:
@d-koppenhagen  I did a` npm pack` and installed the packed version locally

Reply:
@d-koppenhagen it does it for stylesheets in nested directories